Explicitly indicate how object size may be determined (#23949)
mdn/content: files/en-us/web/javascript/reference/global_objects/map/index.md

        {{jsxref("Map.prototype.size", "size")}} property.
      </td>
      <td>

>         Determining the number of items in an <code>Object</code> is more roundabout and less efficient. A common way to do it is by using {{jsxref("Object.keys", "Object.keys")}}
>         and checking the {{jsxref("Array.prototype.length", "length")}} property of the returned array.

Reviewer: ```suggestion
        Determining the number of items in an <code>Object</code> is more roundabout and less efficient. A common way to do it is through the {{jsxref("Array/length", "length")}} of the array returned from {{jsxref("Object.keys()")}}.
```
